packages/react/src/component-tests/IcCard/IcCard.cy.tsx

```diff
@@ -0,0 +1,750 @@
+/// <reference types="Cypress" />
+
+import React from "react";
+import {
+  IcBadge,
+  IcButton,
+  IcCard,
+  IcStatusTag,
+  IcTypography,
+} from "../../components";
+import {
+  BE_VISIBLE,
+  CONTAIN_TEXT,
+  HAVE_CLASS,
+  HAVE_FOCUS,
+  HAVE_LENGTH,
+} from "../utils/constants";
+import { CYPRESS_AXE_OPTIONS } from "../../../cypress/utils/a11y";
+import { mount } from "cypress/react";
+import { SlottedSVG } from "../../react-component-lib/slottedSVG";
+
+const DEFAULT_TEST_THRESHOLD = 0.03;
+
+describe("IcCard", () => {
+  beforeEach(() => {
+    cy.injectAxe();
+  });
+  afterEach(() => {
+    cy.task("generateReport");
+  });
+
+  it("renders Card", () => {
+    mount(
+      <div style={{ margin: "16px" }}>
+        <a href="/">
+          <IcCard heading="Card" message="This is a static card"></IcCard>
+        </a>
+      </div>
+    );
+    cy.checkHydrated("ic-card");
+    cy.get("a").should(HAVE_CLASS, "ic-card-wrapper-link");
+    cy.get("a").focus().should(HAVE_FOCUS);
+    cy.checkA11yWithWait();
+  });
+
+  it("should click on card", () => {
+    mount(
+      <div style={{ margin: "16px" }}>
+        <IcCard
+          id="test-card"
+          heading="Card"
+          message="This is a clickable card"
+          clickable
+        ></IcCard>
+      </div>
+    );
+    cy.checkHydrated("ic-card");
+    cy.get("ic-card").should(BE_VISIBLE).click();
+    cy.compareSnapshot("cardClicked", DEFAULT_TEST_THRESHOLD + 0.03);
+    cy.checkA11yWithWait();
+  });
+
+  it("should get focussed class applied when parent link focussed", () => {
+    mount(
+      <a href="/">
+        <IcCard heading="Card" message="This is a static card"></IcCard>
+      </a>
+    );
+    cy.checkHydrated("ic-card");
+    cy.get("a").focus().should(HAVE_FOCUS);
+    cy.get("ic-card").shadow().find("div").focused();
```

Comment:
There doesn't seem to be an assertion on this line, the .focused() method yields the currently focused DOM element (https://docs.cypress.io/api/commands/focused#Yields)